tests/testapp/tests/test_command.py

```diff
@@ -0,0 +1,32 @@
+from tempfile import NamedTemporaryFile
+
+from django.core.management import call_command
+from django.test import TestCase
+
+
+class TestCommand(TestCase):
+    def test_graph_transitions(self):
+        with NamedTemporaryFile(suffix='.png') as tmpfile:
+            call_command('graph_transitions', 'testapp.Application',
+                         outputfile=tmpfile.name)
+
+        # XXX: collects also all models from the tests, and breaks?!
+        # call_command('graph_transitions', 'testapp')
```

Comment:
Any idea about what is happening here?
Should it work with all the models from the tests, and/or can we make it behave the same way as when it is run from the testapp manually?